components/Modal/Modal.styled.ts

```diff
@@ -17,12 +17,15 @@ export const Background = styled.div`
     rgba(14, 16, 60, 0.6),
     rgba(14, 16, 60, 0.67)
   );
+  display: flex;
+  justify-content: center;
+  align-items: center;
 `;
 
 export const ModalBox = styled(MaxWidth)`
   display: flex;
   flex-direction: column;
-  margin-top: 15vh;
+  margin-bottom: 28vh;
```

Comment:
This is not really reliable because different screen sizes will adjust the margin-bottom. I'd recommend having the align-items at flex-start in the parent container, and changing this to a margin-top instead.